benchmark/sparse/linear_regression/keras_sparse_model.py

```diff
@@ -7,12 +7,14 @@
 from keras import Model
 from keras.layers import Dense, Input
 from keras.optimizers import SGD
+from keras import backend as K
 
 
 def run_benchmark(train_data, train_label, eval_data, eval_label, batch_size, epochs):
-    inputs = Input(shape=(10,), sparse=True)
+    inputs = Input(shape=(10,), dtype='float32', sparse=True)
     predictions = Dense(10, activation='linear')(inputs)
-    model = Model(input=inputs, output=predictions)
+    model = Model(outputs=predictions, inputs=inputs)
```

Comment:
keep it the original order for readability.

Reply:
This change is taking into account the new Keras-2 api for Model

Reply:
I mean to put inputs arguments before outputs argument, they should be equivalent right? https://keras.io/models/model/

Reply:
Oh the ordering, yea I can change that.